ComposeStarter/app/src/main/java/com/example/android/wearable/composestarter/presentation/MainActivity.kt

```diff
@@ -104,21 +104,22 @@ fun WearApp() {
 fun GreetingScreen(greetingName: String, onShowList: () -> Unit) {
     val scrollState = ScrollState(0)
 
-    // Wear design guidelines specify a 5.2% horizontal padding on each side of the list.
-    val horizontalPadding = (0.052f * LocalConfiguration.current.screenWidthDp).dp
```

Comment:
@garanj is removing this and using the SLC padding a net improvement?

Reply:
I would reckon yes? Removes the need for developers to know magic numbers? wdyt?